example/App.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+import React, {useState, useEffect} from 'react';
+import {
+  ScrollView,
+  Button,
+  TextInput,
+  StyleSheet,
+  PermissionsAndroid,
+  Platform,
+  View,
+  Permission,
+} from 'react-native';
+
+import {useJellyfishClient} from '@jellyfish-dev/react-native-client-sdk';
+
+import {Room} from './src/Room';
+import {JELLYFISH_URL} from '@env';
+
+function App(): JSX.Element {
+  const {connect, join, cleanUp} = useJellyfishClient();
+  const [isConnected, setIsConnected] = useState(false);
+  const [peerToken, onChangePeerToken] = React.useState('Peer token');
+
+  useEffect(() => {
+    async function request() {
+      if (Platform.OS === 'ios') {
+        return;
+      }
+      try {
+        const granted = await PermissionsAndroid.requestMultiple([
+          PermissionsAndroid.PERMISSIONS.CAMERA as Permission,
+          PermissionsAndroid.PERMISSIONS.RECORD_AUDIO as Permission,
+        ]);
+        if (
+          granted[PermissionsAndroid.PERMISSIONS.CAMERA as Permission] ===
+            PermissionsAndroid.RESULTS.GRANTED &&
+          granted[PermissionsAndroid.PERMISSIONS.RECORD_AUDIO as Permission] ===
+            PermissionsAndroid.RESULTS.GRANTED
+        ) {
+          console.log('You can use the camera');
+        } else {
+          console.log('Camera permission denied');
+        }
+      } catch (err) {
+        console.warn(err);
+      }
+    }
+
+    request();
+  }, []);
+
+  const connectToRoom = async () => {
+    try {
+      await connect(JELLYFISH_URL, peerToken);
+      setIsConnected(true);
+      await join({name: 'RN mobile'});
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const disconnect = async () => {
+    cleanUp();
+    setIsConnected(false);
+  };
+
+  return (
+    <View style={styles.app}>
+      {isConnected ? (
+        <View style={[styles.button, styles.disconnectButton]}>
+          <Button title="Disconnect" onPress={disconnect} />
+        </View>
+      ) : (
+        <View style={styles.noCallBody}>
+          <TextInput
+            style={styles.input}
```

Comment:
A little quality of life change: make a placeholder here instead of initial text, I don't want to delete "peerToken" every time I want to enter the peer token
Also on my phone the text is white on light blue at that looks very ugly and unreadable. I know this isn't supposed to be pretty app but it should be at least readable :D

Reply:
Done.